src/main/java/worldofzuul/EnergySources/EnergySource.java

```diff
@@ -22,6 +28,21 @@ public void setEnergyName(String energyName) {
     }
 
     public int getEnergyPrice() {
+        Game.instance.getGameYear();
+
+        if(energyEmission <= 0){
+            Random rand = new Random();
+            double random_int = rand.nextInt(5 - 2 + 1) + 2/100;
+            double procent = (random_int/100);
+            int fivePercent = (int)(energyPrice * procent);
```

Comment:
This percentage functionality is duplicated, we could extract it to a method, and specify the percentage as a parameter.